### PR TITLE
error fixed on line 419 ch12_part2.ipynb

### PR DESCRIPTION
**Pull request by vg11072001**
Hi Sebastian & team,
I was going through the PyTorch notebook and found one error. It was caused by the repetition of the use of `.detach().numpy()`, which needs to be removed from one place. 
Thanks & warm regards
Vanshika

PS. your book is great for basic concepts

### CODE REVIEW

**Comment by rasbt** [comment]
Thanks, this fix looks good. Not sure how this code ran before. Maybe a PyTorch 1.0 thing. Anyways, thanks for submitting!
